### PR TITLE
Refactor: Made landing and docs page responsive

### PR DESCRIPTION
**Pull request by LordUttam**
Made landing and docs pages responsive

image
image

### CODE REVIEW

**Review by NeerajVodala — APPROVED** [review]
Looks good to me.
All the best.

File: index.html [hunk 1]
```diff
@@ -57,13 +57,16 @@ <h2>Installation</h2>
                 <p>Simply copy and paste the code in the head tag of your html document.</p>
                 <a href="/pages/docs.html"><button class="btn btn--primary">See components</button></a>
             </div>
-            <div class="right">
+            <div class="right install-div--large">
                 <iframe
                     src="https://carbon.now.sh/embed?bg=rgba%28216%2C225%2C255%2C1%29&t=theme%3Aap7qpqk54lo&wt=none&l=htmlmixed&width=633.0625&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=56px&ph=56px&ln=false&fl=1&fm=Fira+Code&fs=13.5px&lh=163%25&si=false&es=2x&wm=false&code=%253C%21--%2520Copy%2520and%2520paste%2520this%2520in%2520the%2520%253Chead%253E%2520of%2520your%2520html.%2520--%253E%250A%253Clink%2520rel%253D%2522stylesheet%2522%2520href%253D%2522https%253A%252F%252Fkalpaui.netlify.app%252Fstyles%252Fmain.css%2522%253E"
                     style="width: 633px; height: 235px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (bharati-21) [inline index.html:63]: Kindly avoid inline styles.

File: styles/index.css [hunk 2]
```diff
@@ -204,6 +204,10 @@ h2 {
   box-shadow: var(--primary-box-shadow);
 }
 
+.install-div--mobile {
+  display: none;
+}
+
 .btn--outline:hover {
   background-color: rgba(0, 0, 0, 0.2);
```
Comment (bharati-21) [inline styles/index.css:212]: You could be consistent and use either variables everywhere or color values rgba/hex/name

File: styles/index.css [hunk 3]
```diff
@@ -277,3 +281,97 @@ h2 {
...
+  .main__text,
+  #installation {
+    margin: 0 1rem;
+  }
+  #installation {
+    flex-wrap: wrap;
+    height: 63vh;
+  }
+
+  #installation {
```
Comment (bharati-21) [inline styles/index.css:305]: I see repeated styles. Why not combine them in a single file?

File: javascript/mobile-menu.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+const mobileMenu = document.querySelector(".menu--mobile");
+const sidebarBtn = document.querySelector(".sidebar-btn");
```
Comment (bharati-21) [inline javascript/mobile-menu.js]: Seems like you have a class naming that does not follow BEM convention.
Makes sense to name it `sidebar__btn`

File: pages/docs.html [hunk 5]
```diff
@@ -1221,11 +1281,16 @@ <h1 class="main__heading p--1" id="snackbar">
         </main>
     </div>
 
+    <button class="btn--float sidebar-btn">
```
Comment (bharati-21) [inline pages/docs.html]: You can look into sidebar-btn here and ensure you follow consistent BEM.

**Review by bharati-21 — COMMENTED** [review]
I see repeated styles in docs.css and index.css. Dropped a few suggestions. If you have the bandwidth you can work on it.

File: index.html (hunk 1 above)
Comment (LordUttam) [reply index.html:63]: Sorry but I think those are from carbon's iframe

File: pages/docs.html (hunk 5 above)
Comment (LordUttam) [reply pages/docs.html]: yeah I was thinking about that. I have named it menu__btn now. does that work?

File: index.html (hunk 1 above)
Comment (bharati-21) [reply index.html:63]: > Sorry but I think those are from carbon's iframe

You can add a class for it, copy, and paste it in your stylesheet.

File: pages/docs.html (hunk 5 above)
Comment (bharati-21) [reply pages/docs.html]: > yeah I was thinking about that. I have named it menu__btn now. does that work?

Looks good!

**Pull request merged**